### PR TITLE
[bulk] Validation: metadata and tab group changes

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

**Comment by v-emilypr** [comment]
**Global effort to fix build validation errors**

@Lauragra, please see the proposed changes and let me know if anything should be addressed differently. Thank you! 

@andrueastman the following file contains a tab group error: api-reference/v1.0/api/securescorecontrolprofile-update.md

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/v1.0/api/group-delete.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.prod: "groups"
 doc_type: apiPageType
 ---
 
-# Delete group
+# Delete group - Microsoft Graph API
```
Comment (Lauragra) [inline api-reference/v1.0/api/group-delete.md:10]: Global: Please remove. We haven't agreed to differentiate our topic titles; we might need to request to have this rule turned off on our repo.

File: api-reference/v1.0/api/shares-get.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ ms.date: 09/10/2017
 title: Access shared items
 ms.localizationpriority: medium
 description: "Access a shared DriveItem or a collection of shared items by using a shareId or sharing URL."
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/shares-get.md:7]: files

File: api-reference/v1.0/api/sites-list-followed.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ author: learafa
 description: "List the sites that have been followed by the signed in user."
 title: List followed sites
 ms.localizationpriority: medium
-ms.prod: "SharePoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/v1.0/api/sites-list-followed.md:6]: sites-and-lists

File: api-reference/v1.0/api/shares-get.md (hunk 2 above)
Comment (v-emilypr) [reply api-reference/v1.0/api/shares-get.md]: change made

File: api-reference/v1.0/api/sites-list-followed.md (hunk 3 above)
Comment (v-emilypr) [reply api-reference/v1.0/api/sites-list-followed.md]: change made

**Review by Lauragra — APPROVED** [review]

**Pull request merged**